src/addMeToYourFriends.ts

```diff
@@ -0,0 +1,146 @@
+import { html, TemplateResult } from "lit-html";
+import { styleMap } from "lit-html/directives/style-map";
+import { DataBrowserContext, LiveStore } from "pane-registry";
+import * as UI from "solid-ui";
+import { padding, textCenter } from "./baseStyles";
+
+let buttonContainer = <HTMLDivElement>document.createElement("div");
+//TODO create dedicated component in UI
+const positiveFrontendMessageDiv = <HTMLDivElement>document.createElement("div");
+positiveFrontendMessageDiv.setAttribute(
+  "style",
+  "margin: 0.1em; padding: 0.5em; border: 0.05em solid gray; background-color: #efe; color:black;"
+);
+
+const styles = {
+  button: styleMap({ ...textCenter(), ...padding() }),
+};
+
+//Same not logged in error message like on 'Chat with me' buton
+const userNotLoggedInErrorMessage = "Current user not found! Not logged in?";
+const internalErrorMessage = "An internal error occured!";
+const friendWasAddedSuccesMessage = "Friend was added!";
+const friendExistsMessage = "Friend already exists";
+const addMeToYourFriendsButtonText = "Add me to your friend list";
+const friendExistsAlreadyButtonText = "Already part of friend list";
+
+const addMeToYourFriendsHtml = (

```

Comment:
Suggestion: I don't think you need to have the html.  you might call it the thing it is like addMeToYourFriendsButton perhaps.

Reply:
I call it Div instead of Html